src/app/modules/brightness.cpp

```diff
@@ -16,7 +16,7 @@ void MockedBrightnessModule::set_brightness(int brightness) { this->window->setW
 
 RpiBrightnessModule::RpiBrightnessModule() : BrightnessModule(false), brightness_attribute(this->PATH)
 {
-    this->brightness_attribute.open(QIODevice::WriteOnly | QIODevice::ExistingOnly);
+    this->brightness_attribute.open(QIODevice::WriteOnly| QIODevice::ExistingOnly);
```

Comment:
In develop I just removed the ExistingOnly since it wasn't really needed (more of an extra precaution)

So you'll probs have a conflict here but just go ahead and remove it

Reply:
Haha, we've all done that